concurrent-ruby-edge.gemspec

```diff
@@ -23,7 +23,7 @@ be obeyed though. Features developed in `concurrent-ruby-edge` are expected to m
 Please see http://concurrent-ruby.com for more information.
   TXT
 
-  s.required_ruby_version = '>= 1.9.3'
+  s.required_ruby_version = '>= 2.2.10'
```

Comment:
Should be just `2.2` as Jeremy said, it's very uncommon to specify 3rd component for required_ruby_version, and this could be a problem for an alternative implementation which might not use the 3rd component in the same or it might not matter (unlikely for this case since 2.2 is old, but still)